go/client/chat_cli_rendering.go

```diff
@@ -354,22 +354,7 @@ type messageView struct {
 	messageType chat1.MessageType
 }
 
-// newMessageView extracts from a message the parts for display
-// It may fetch the superseding message. So that for example a TEXT message will show its EDIT text.
-func newMessageView(g *libkb.GlobalContext, conversationID chat1.ConversationID, m chat1.MessageUnboxed) (messageView, error) {
-
-	mv := messageView{}
-	st, err := m.State()
-	if err != nil {
-		return mv, fmt.Errorf("unexpected empty message")
-	}
-	if st == chat1.MessageUnboxedState_ERROR {
-		return mv, fmt.Errorf("<%s>", m.Error().ErrMsg)
-	}
-	// TODO handle messages in outbox properly
-	if st == chat1.MessageUnboxedState_OUTBOX {
-		return mv, fmt.Errorf("<message waiting to be sent>")
-	}
+func newMessageViewValid(g *libkb.GlobalContext, conversationID chat1.ConversationID, m chat1.MessageUnboxed) (mv messageView, err error) {
```

Comment:
This should take `m chat1.MessageUnboxedValid` now instead of `m chat1.MessageUnboxed`.